### PR TITLE
VAR-138 | Add Edit, Cancel, Deny, Approve buttons to Manage Reservation

### PR DESCRIPTION
**Pull request by midori0507**

### CODE REVIEW

File: app/i18n/messages/en.json [hunk 1]
```diff
@@ -314,6 +314,7 @@
   "ManageReservationsList.actionLabel.approve":"Approve",
   "ManageReservationsList.actionLabel.deny":"Deny",
   "ManageReservationsList.actionLabel.edit":"Edit",
+  "ManageReservationsList.actionLabel.cancel":"Cancel",
```
Comment (Nizar-Rahme) [inline app/i18n/messages/en.json]: How about other languages?

File: src/domain/reservation/modal/_reservationInformationModal.scss [hunk 2]
```diff
@@ -10,6 +10,9 @@
     }
   }
 
+  &__edit-reservation {
+    float: right;
```
Comment (Nizar-Rahme) [inline src/domain/reservation/modal/_reservationInformationModal.scss]: It's 2019! 😃

File: src/domain/reservation/modal/ReservationInfomationModal.js [hunk 3]
```diff
@@ -44,22 +45,45 @@ const ReservationInfomationModal = ({
...
+          className="app-ReservationInfomationModal__edit-reservation"
+          onClick={() => onEditClick(reservation)}
+        >
+          {t('ReservationEditForm.startEdit')}
+        </Button>
       </Modal.Body>
       <Modal.Footer>
         <Button
           bsStyle="primary"
+          onClick={() => onHide()}
```
Comment (Nizar-Rahme) [inline src/domain/reservation/modal/ReservationInfomationModal.js]:
```suggestion
          onClick={onHide}
```

File: src/domain/reservation/manage/page/ManageReservationsPage.js [hunk 4]
```diff
@@ -176,4 +209,13 @@ class ManageReservationsPage extends React.Component {
 }
 
 export const UnwrappedManageReservationsPage = injectT(ManageReservationsPage);
-export default withRouter(UnwrappedManageReservationsPage);
+
+const mapDispatchToProps = (dispatch) => {
+  const actionCreators = {
+    editReservation: selectReservationToEdit
```
Comment (Nizar-Rahme) [inline src/domain/reservation/manage/page/ManageReservationsPage.js]: Not sure why we still need this!

File: src/domain/reservation/manage/page/ManageReservationsPage.js [hunk 5]
```diff
@@ -13,6 +15,9 @@ import ManageReservationsList from '../list/ManageReservationsList';
 import Pagination from '../../../../common/pagination/Pagination';
 import * as searchUtils from '../../../search/utils';
 import ReservationInfomationModal from '../../modal/ReservationInfomationModal';
+import { selectReservationToEdit } from '../../../../../app/actions/uiActions';
```
Comment (Nizar-Rahme) [inline src/domain/reservation/manage/page/ManageReservationsPage.js]: I'd rather do without this action

File: src/domain/reservation/manage/page/ManageReservationsPage.js (hunk 4 above)
Comment (midori0507) [reply src/domain/reservation/manage/page/ManageReservationsPage.js]: To populate the timeslot. I have a todo to remove them in the future

File: src/domain/reservation/manage/page/ManageReservationsPage.js (hunk 5 above)
Comment (midori0507) [reply src/domain/reservation/manage/page/ManageReservationsPage.js]: The edit reservation view need this to work, they still use redux

**Pull request merged**